Assets/tiny_rogue/Scripts/ArchetypeLibrary.cs

```diff
@@ -177,6 +191,40 @@ public Entity CreateStairway(EntityManager entityManager, int2 xy, float3 pos)
 
             return entity;
         }
+        
+        public Entity CreateCollectible(EntityManager entityManager, int2 xy, float3 pos)
+        {
+            Entity entity = entityManager.CreateEntity(Collectible);
+
+            Sprite2DRenderer s = new Sprite2DRenderer();
+            Translation t = new Translation();
+            WorldCoord c = new WorldCoord();
+            LayerSorting l = new LayerSorting();
+            CanBePickedUp p = new CanBePickedUp();
+            t.Value = pos;
+
+            c.x = xy.x;
+            c.y = xy.y;
+
+            s.color = new Unity.Tiny.Core2D.Color(1, 1, 1);
+            // TODO: need to figure out collectible tile
+            s.sprite = SpriteSystem.IndexSprites[4];
+            l.order = 1;
+
+            p.appearance.sprite = s.sprite;
+            p.appearance.color = s.color;
+            
+            p.name = new NativeString64("sword");
+            p.description = new NativeString64("Sword of Damocles");
+
+            entityManager.SetComponentData(entity, s);
+            entityManager.SetComponentData(entity, t);
+            entityManager.SetComponentData(entity, c);
+            entityManager.SetComponentData(entity, l);
+            entityManager.SetComponentData(entity, p);
+
+            return entityManager.Instantiate(entity);
```

Comment:
This should be `return entity`. It's originally my bug that's just been copied over here. If this is called it will generate the entity in the world twice.